README.md

```diff
@@ -1,340 +1,535 @@
+#### Launch with Docker and YAML file
+All of the necessary configuration information can be provided in a YAML file (referenced as `initiator.yaml` from now on).
 
-#### Start a DKG-operator
+A good way to manage all the necessary files (`operators_info.json`, `encrypted_private_key.json`, `password`) is to store them in a single folder (in this case `initiator-config`) together with the `initiator.yaml` configuration file, like so:
 
 ```sh
-ssv-dkg start-operator \
-            --privKey ./examples/operator1/encrypted_private_key.json  \
-            --port 3030 \
-            --password ./examples/operator1/password \
-            --storeShare true \
-            --logLevel info \
-            --logFormat json \
-            --logLevelFormat capitalColor \
-            --logFilePath ./examples/output/operator1_logs_debug.log
-
-### where
---privKey ./examples/operator1/encrypted_private_key.json # path to ssv operator`s private key
---port 3030 # port for listening messages
---password: ./examples/operator1/password # path to password file to decrypt the key
---storeShare: true # store created bls key share to a file for later reuse if needed
---logLevel: info # logger's log level (info/debug/
---logFormat: json # logger's encoding, valid values are 'json' (default) and 'console'
---logLevelFormat: capitalColor # logger's level format, valid values are 'capitalColor' (default), 'capital' or 'lowercase''
---logFilePath: ./examples/operator1_logs_debug.log # a file path to write logs into
+ssv@localhost:~/ssv-dkg# tree initiator-config
+initiator-config
+├── encrypted_private_key.json
+├── initiator.yaml
+├── operators_info.json
+└── password
+
+1 directory, 4 files
 ```
 
-Its also possible to use yaml configuration file `./config/operator.yaml` for parameters. `ssv-dkg` will be looking for the config file at `./config/` folder.
-
-Example:
+With this configuration, a typical configuration file would look like this:
 
 ```yaml
-privKey: /data/operator1/encrypted_private_key.json
-password: /data/operator1/password
-port: 3030
-storeShare: false
-logLevel: info
-logFormat: json
-logLevelFormat: capitalColor
-logFilePath: /data/output/operator1_logs_debug.log
+operatorIDs: [143, 219, 33, 34]    # array of Operator IDs which will be used for a DKG ceremony
+withdrawAddress: "0xa1a66cc5d309f19fb2fda2b7601b223053d0f7f4"    # Address where reward payments for the validator are sent
+owner: "0xb64923DA2c1A9907AdC63617d882D824033a091c"    # Address of owner of the Cluster that will manage the validator on ssv.network
+nonce: 0    # Owner nonce for the SSV contract
+network: "prater"    # Network name (default: mainnet)
+operatorsInfoPath: /data/operators_info.json    # Path to the file containing operators information
+# alternatively:
+# operatorsInfo: '{ 1: { publicKey: XXX, id: 1, ip: 10.0.0.1:3033 }'    # Raw content of the JSON file with operators information
```

Comment:
this was originally a comment by me but it's better to provide to correct format here.

Reply:
from current readme
```bash
--operatorsInfo '[{"id": 1,"public_key": "LS0tLS1CRUdJTiBSU0....","ip": "http://localhost:3030"}, {"id": 2,"public_key": "LS0tLS1CRUdJTiBSU0....","ip": "http://localhost:3030"},...]' # raw JSON string containing operators info.
```